APIManager.js

```diff
@@ -0,0 +1,67 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+
+    getData() {
```

Comment:
A better practice would be to separate each API call to a different method (`getQuote`, `getPokemon`, etc.) and then create one method that invokes all the other functions